build.gradle

```diff
@@ -139,21 +127,10 @@ allprojects {
     }
 
     repositories {
-        mavenLocal()
```

Comment:
mavenLocal is used when you build library with a locally built compiler, so I suggest to leave it at the end of the list